Packs/AWS-SNS-Listener/ReleaseNotes/1_0_4.md

```diff
@@ -0,0 +1,7 @@
+
+#### Integrations
+
+##### AWS-SNS-Listener
+
+- Fixed an issue with response type.
```

Comment:
Maybe it should be something like this:
- Fixed an issue where the status response type was incorrect.